server.js

```diff
@@ -2,7 +2,22 @@ const express = require('express'),
       app = express(),
       router = express.Router(),
       fs = require('fs'),
-      bodyParser = require('body-parser');
+      bodyParser = require('body-parser')
+      mongoose = require('mongoose')
+      url = 'mongodb://localhost:27017',
+      dbName = 'news';
+
+mongoose.connect(`${url}/${dbName}`, { useNewUrlParser: true });
+
+const newsSchema = new mongoose.Schema({
+    id: Number,
+    author: String,
+    title: String,
+    description: String,
+    content: String
+});
```

Comment:
Perhaps, it should be moved to the models, could you please restructure the server-side?